readme.md

```diff
@@ -41,6 +41,22 @@ cd app
 make
 ```
 
-## Note on ST-Link
+## Debuggers
 
-The repository does not yet include instructions or tools for working with the ST-Link. If you get the ST-Link up and running, please submit a PR with a *separate* .vscode directory (`.vscode-stlink`) for tasks and launch configurations, along with usage instructions for the readme. Your help will be very much appreciated!
```

Comment:
I guess I'll proactively note that the README requested that the st-link configuration be placed in a separate .vscode-stlink folder. I couldn't figure out how to configure VSCode to use any directory not named .vscode and since there's so much overlap and it's not hard to support both in the same .vscode directory I opted for this merged structure.

If that's truly not desirable, I can split it out but I thought I'd start with this